torchvision/prototype/transforms/_type_conversion.py

```diff
@@ -14,6 +14,7 @@ class DecodeImage(Transform):
     _transformed_types = (features.EncodedImage,)
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> features.Image:
+        # FIXME: should we move this wrapping into the functional?
```

Comment:
Let's fix these fixme here ?

Reply:
I would prefer that as stated above. Let's see if we all agree with the proposals. I'll send a new commit after.

Reply:
This was done by @datumbox in #6502.

Reply:
I didn't modify `decode_image_with_pil()`. Should I?

Reply:
I've done it on #6502.
```suggestion
```